projects/azure/image-mirror/image-mirror.yaml

```diff
@@ -10,6 +10,7 @@ items:
   data:
     openshift-azure_v3_10_quay: |
       registry.svc.ci.openshift.org/azure/azure-plugins:sync quay.io/openshift-on-azure/sync:v3.10 quay.io/openshift-on-azure/sync:v3.10.0 quay.io/openshift-on-azure/sync:latest
+      registry.svc.ci.openshift.org/azure/azure-misc:prometheus-sidecar quay.io/openshift-on-azure/prometheus-sidecar:v3.10 quay.io/openshift-on-azure/prometheus-sidecar:v3.10.0 quay.io/openshift-on-azure/prometheus-sidecar:latest
```

Comment:
Do you have a PR somewhere for adding the azure-misc imagestream?

Reply:
why would we have a prometheus-sidecar:v3.10 ?

Reply:
it just matches versioning of all other. latest only?